EIPS/eip-5883.md

```diff
@@ -0,0 +1,110 @@
+---
+eip: 5883
+title: Token Transfer by Social Recovery
+description: On-Chain Social Recovery
```

Comment:
Oops, forgot to also mention that the description needs to be expanded. Try to mention what makes your proposal stand out compared to other social recovery methods.